packages/base-manager/package.json

```diff
@@ -35,14 +35,16 @@
     "@jupyter-widgets/base": "^4.0.0-alpha.0",
     "@jupyterlab/services": "^5.0.2",
     "@lumino/coreutils": "^1.4.2",
-    "base64-js": "^1.2.1"
+    "base64-js": "^1.2.1",
+    "sanitize-html": "^1.20"
```

Comment:
I think it was what we were familiar with (partly because we chose it for jlab long ago). I think if we want to switch to dompurify, we can do it in a follow-up PR.